src/main/java/edu/wisc/my/rssToJson/dao/RssToJsonDaoImpl.java

```diff
@@ -31,36 +42,81 @@
     @Autowired
     void setEnv(Environment env) {
       this.env = env;
+    }   
+    private String httpResponseAsString(String url) throws IOException {
+        logger.error("HTTP Response method " + url);
+        CloseableHttpClient httpclient = HttpClients.createDefault();
+        try {
+            HttpGet httpget = new HttpGet(url);
+            logger.error(httpget.toString());
+
+            // Create a custom response handler
+            ResponseHandler<String> responseHandler = new ResponseHandler<String>() {
+
+                @Override
+                public String handleResponse(
+                        final HttpResponse response) throws ClientProtocolException, IOException {
+                     logger.error("TWO " + response.toString());        
+                    int status = response.getStatusLine().getStatusCode();
+                    logger.debug(status + " response code ");
+                    if (status >= 200 && status < 300) {
+                        HttpEntity entity = response.getEntity();
+                        return entity != null ? EntityUtils.toString(entity) : null;
+                    } else {
+                        throw new ClientProtocolException("Unexpected response status: " + status);
+                    }
+                }
+            };
+            String responseBody = httpclient.execute(httpget, responseHandler);
+            logger.error("ONE POINT SIX " + responseBody);
+            return responseBody;
+        } finally {
+            httpclient.close();
+        }
     }
 
+
+    @Override
+    public JSONObject getXMLFeed(String feedEndpoint) {
+       String endpointURL = getEndpointURL(feedEndpoint);
+       JSONObject jsonObject = null;
+       try {
+         String xmlString = httpResponseAsString(endpointURL);
+         jsonObject = XML.toJSONObject(xmlString);
+       } catch (Exception e) {
+           logger.warn(e.getMessage());
+           return null;
+       }
+       return jsonObject;
+   }
+   
+   public String getEndpointURL(String feed) {
+       logger.error("GETTING THE ENDPOINT FOR " + feed);
```

Comment:
Some of these logging levels are too high, it looks like these should be debug logs, not error